app/models/dialog_field_serializer.rb

```diff
@@ -8,6 +12,15 @@ def initialize(resource_action_serializer = ResourceActionSerializer.new)
   def serialize(dialog_field)
     serialized_resource_action = @resource_action_serializer.serialize(dialog_field.resource_action)
 
-    included_attributes(dialog_field.attributes).merge("resource_action" => serialized_resource_action)
+    extra_attributes = {
+      "resource_action" => serialized_resource_action
+    }
+
+    if dialog_field.dynamic?
+      dynamic_values = dialog_field.trigger_automate_value_updates
+      extra_attributes.merge!("values" => dynamic_values)
```

Comment:
`extra_attributes["values"] = dynamic_values` - has the same effect, but shorter and faster